benchmarks/common.py

```diff
@@ -1212,8 +1212,9 @@ def run_performance_test(
             optimized_model_iter_fn = optimize_ctx(self.model_iter_fn)
 
             try:
-                torch.cuda.reset_peak_memory_stats()
-                torch.cuda.empty_cache()
+                if self.args.devices == ["cuda"]:
```

Comment:
Maybe use `current_device`?

Reply:
It makes sense. The way how `args.devices` is currently used in other places may need clean up as well, but it's low priority as we don't really mix multiple device measurement.